python/src/aac/validate/_validate.py

```diff
@@ -42,36 +42,33 @@ def validated_source(source: str) -> Generator[ValidationResult, None, None]:
     yield _with_validation(parse(source))
 
 
-def _with_validation(user_definitions: list[Definition]) -> ValidationResult:
+def _with_validation(user_definitions: list[Definition]) -> ValidatorResult:
     try:
         result = _validate_definitions(user_definitions)
 
-        if result.is_valid:
+        if result.is_valid():
             return result
         else:
-            raise ValidationError("Failed to validate content with errors:", *result.messages)
+            raise ValidationError("Failed to validate content with errors:", result.get_messages_as_string())
```

Comment:
Looks like there is a redundant error message in the result that we can remove:

Old output
```
validate: validation_failure

Failed to validate:

Failed to validate content with errors:
```

New output
```
validate: validation_failure

Failed to validate:
```

```suggestion
            raise ValidationError(result.get_messages_as_string())
```